sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestBase.java

```diff
@@ -29,9 +43,17 @@
     // Environment variable name used to determine the TestMode.
     private static final String AZURE_TEST_MODE = "AZURE_TEST_MODE";
     private static final String AZURE_TEST_HTTP_CLIENTS = "AZURE_TEST_HTTP_CLIENTS";
-    public static final String AZURE_TEST_HTTP_CLIENTS_VALUE_ALL = "ALL";
-    public static final String AZURE_TEST_HTTP_CLIENTS_VALUE_NETTY = "NettyAsyncHttpClient";
+    private static final String AZURE_TEST_HTTP_CLIENTS_VALUE_ALL = "ALL";
+    private static final String AZURE_TEST_HTTP_CLIENTS_VALUE_ROLLING = "rolling";
+    private static final String AZURE_TEST_HTTP_CLIENTS_VALUE_NETTY = "netty";
     public static final String AZURE_TEST_SERVICE_VERSIONS_VALUE_ALL = "ALL";
+    public static final String AZURE_TEST_SERVICE_VERSIONS_VALUE_ROLLING = "rolling";
+    public static final int PLATFORM_COUNT = 6;
```

Comment:
Since we are using the file to read the list of pipelines, the count should be the size of the list instead of hardcoding it to 6.